gold-cc-cvc-input.html

```diff
@@ -67,6 +67,10 @@
         margin-left: 10px;
       }
 
+      iron-icon[hidden] {
```

Comment:
I think that one is fine, since it doesn't have a `display: block` or anything like that anywhere